Makefile

```diff
@@ -88,14 +93,21 @@ set-saas-configuration:
 start-runner:
 	docker build -t quickstart-runner -f build/runner.dockerfile .
 	docker-compose -f docker-compose.acp.local.yaml up -d runner
+	docker ps -a
+
+.PHONY: stop-runner
+stop-runner:
+	docker rm -f quickstart-runner
 
 .PHONY: generate-obuk-integration-spec
 generate-obuk-integration-spec: start-runner
 	./scripts/generate_bank_spec.sh uk
+	make stop-runner
```

Comment:
why this is necessary? I'd remove it here and leave only in necessary places

Reply:
ok